Luminary099/STABLE_ORBIT.agc

```diff
@@ -202,7 +202,7 @@ RECYCLE		CALL
 		PDDL	DDV
 			DELTAR
 		SL*			# DELTA R / (VP X RP/RP)
-			0 	-7,1
+			0 	-7.1
```

Comment:
Sorry, I answered your query on this on my phone. Counter to what I originally said this is supposed to be _yet another_ badly printed comma. 
![image](https://user-images.githubusercontent.com/3440094/84194250-a8143600-aa94-11ea-8d4a-97eb87f0478c.png)

You can see just how inconsistently `,` was printed on this page by looking at the 6th column of the printout.

![image](https://user-images.githubusercontent.com/3440094/84195476-8ddb5780-aa96-11ea-92b0-a1a2daf1cb98.png)
 
```suggestion
			0 	-7,1
```